larq/layers.py

```diff
@@ -75,6 +75,8 @@ class QuantDense(QuantizerBase, tf.keras.layers.Dense):
         the output of the layer (its "activation").
     kernel_constraint: Constraint function applied to the `kernel` weights matrix.
     bias_constraint: Constraint function applied to the bias vector.
+    metrics: An array of metrics to add to the layer. If `None` the metrics set in
+        `larq.metrics.scope` are used. Available metrics: `flip_ration`.
```

Comment:
```suggestion
        `larq.metrics.scope` are used. Available metrics: `flip_ratio`.
```

Would it make sense to make this a hyperlink?

Reply:
Currently we don't have a documentation page for `flip_ratio` since I am still unsure if we want to change the API of the `FlipRatio` class in the future. I think it is fine to keep it as is and we can add more documentation on metrics when we add more.